src/unity/lib/annotation/README.md

```diff
@@ -35,7 +35,7 @@ https://github.com/protocolbuffers/protobuf/releases/tag/v3.3.0
 When you've installed the package, regenerate these protobuf files use the command below in the current directory.
 
 ```bash
-protoc --proto_path="./format" --cpp_out="./build/format/cpp" "./format/annotate.proto" "./format/data.proto" "./format/meta.proto"
+protoc --proto_path="./format" --cpp_out="./build/format/cpp" "./format/annotate.proto" "./format/data.proto" "./format/message.proto" "./format/meta.proto"
```

Comment:
Doesn't need to be addressed in this commit, but it should be possible to put a build rule in CMakeLists.txt that doesn't run by default (so regular `make` won't invoke it) but we could manually run something like `make protosrc` to regenerate.

Reply:
Noted! Thanks!